lib/configParser.js

```diff
@@ -33,6 +33,7 @@ var ConfigParser = function() {
       defaultTimeoutInterval: (30 * 1000)
     },
     seleniumArgs: [],
+    sessionId: null,
```

Comment:
Can you also update our master list of configuration docs at https://github.com/angular/protractor/blob/master/docs/referenceConf.js?